gammapy/spectrum/tests/test_flux_point.py

```diff
@@ -39,7 +41,9 @@ def inverse(self, y):
 
 
 class XSqrTestModel(SpectralModel):
-    parameters = Parameters([])
+    def __init__(self):
+        self._parameters = Parameters([])
```

Comment:
@adonath - how about making the default `None` and to mean to create an empty container?
Then you can write `Parameters()` everywhere instead of `Parameters([])`.
That's common, see e.g. `dict()`, `list()`, `set()`.
Should I send a PR with this change, and simplify existing callers?

Reply:
I think it's actually OK as is, because `Parameters` are not build programmatically
such as `list` or `dict`. So the case of instantiating empty `Parameters` is really
rare...but I don't oppose either. Go ahead if you like...

Reply:
I think every user that defines a model has to write `([])`, no?
I'll send a PR ...